src/express-validator.spec.ts

```diff
@@ -183,4 +183,19 @@ describe('High level tests', () => {
     const result = await body('foo.*.nop').exists().run(req);
     expect(result.isEmpty()).toEqual(true);
   });
+  it('should error array if no wildcard is used', async () => {
```

Comment:
Similar feeling to https://github.com/express-validator/express-validator/pull/1279#discussion_r1501438112.